src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1711,6 +1737,7 @@ def forward(
                 class_embeds,
                 text_outputs.to_tuple(),
                 vision_outputs.to_tuple(),
+                objectness_logits,
```

Comment:
this is breaking 😅 
One way to prevent this from breaking is to have a `return_objectness_logits` set to false by default